src/Foundation/NSObject.mac.cs

```diff
@@ -42,6 +42,7 @@ public partial class NSObject : INativeObject
 		// note: the linker will remove the unrequired `dlopen` calls
 		// Used to force the loading of AppKit and Foundation
 		// ** DO NOT rename the fields as mmp's linker can remove them when not required **
+		// ** IF YOU ADD ITEMS HERE PLEASE UPDATE linker/ObjCExtensions.cs and mmp/linker/MonoMac.Tuner/MonoMacNamespaces.cs
```

Comment:
^ not just for the committer but also people reviewing a PR will see the reminder :)